{"name": "Wrong user password error test", "status": "passed", "start": 1727809291569, "stop": 1727809291741, "uuid": "92e4a2f4-1374-46d7-8ceb-5fc15a5d4c13", "historyId": "799edbf56d1db223e36e2e4d7f0441bb", "testCaseId": "799edbf56d1db223e36e2e4d7f0441bb", "fullName": "tests.test_user_login.TestUserLogin#test_user_login_with_wrong_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "WIN-B95O9QQ3RDO"}, {"name": "thread", "value": "23340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}